Lesson 2/app/src/main/res/layout/activity_account.xml

```diff
@@ -9,110 +9,14 @@
     tools:context="com.mobileapps.lesson2.AccountActivity"
     tools:ignore="ExtraText">
 
```

Comment:
activity_account is a little ambiguous name. Is it details of the account you want to see as an user? Of a credit account? Or user account?